Makefile

```diff
@@ -1116,6 +1116,15 @@ $(SQLPARSER_ROOT)/help_messages.go: $(SQLPARSER_ROOT)/sql.y $(SQLPARSER_ROOT)/he
 	mv -f $@.tmp $@
 	gofmt -s -w $@
 
+CLANG_FORMAT := $(shell which clang-format)
```

Comment:
@benesch: my make-fu is rather weak. Could you take a quick look at this new rule?

Reply:
Looks good! (We don't usually bother to check whether tools are installed, FWIW, and just live with the `/bin/bash: BLAH: command not found` error.)